src/App.js

```diff
@@ -40,8 +39,7 @@ const App = () => {
         const unregister = chrome.on('APP_NAVIGATION', event => {
             chrome.hideGlobalFilter(false);
             if (event.domEvent) {
-                push(`/${event.navId}`);
-                appNavClick[event.navId] !== undefined ? appNavClick[event.navId](true) : appNavClick.cves(true);
+                chrome.appNavClick(`/${event.navId}`);
             }
         });
```

Comment:
This whole part related to `APP_NAVIGATION` can be removed.

Reply:
Done. Also removed it in the UpgradePage.js file.